examples/medplum-hello-world/src/components/PatientHistory.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { usePatientInfo } from '../pages/PatientPage';
+import { Document, ResourceHistoryTable } from '@medplum/react';
+
+export function PatientHistory(): JSX.Element {
+  const { patient } = usePatientInfo().data;
+  return (
+    <Document>
+      <h3>Patient History</h3>
```

Comment:
There would be a lot of added complexity with adding an underline over the tab due to the constant re-rendering. I figure it'll be simpler to add a title to each of the new components